iocage_lib/ioc_list.py

```diff
@@ -455,6 +461,25 @@ def list_all(self, jails):
                 jail_list.append([jid, uuid, boot, state, jail_type,
                                   full_release, full_ip4, ip6, template,
                                   admin_portal, doc_url])
+                if self.plugin_data:
```

Comment:
We should probably want the remote flag set for this to occur, as list is expecting all resources to be listed are local and quick. This could have a long delay if any network disruptions are occurring or anything else is funky network wise.

Reply:
Are you referring to `iocage list` ?

Reply:
Yeah if this is ever exposed to cli behavior.

Reply:
I don't think that it will have much if any use there. Also we could be perhaps increasing more props here, so the formatting is not going to work at all. I doubt it will even with just 3 more props. Plus the repo uri's are quite big ; ) I think we are good for now. However wrt the PR, i am thinking we should do the loop asynchronously, will look more into that tomorrow. Marking as WIP :p